src/components/10-atoms/input-text/index.js

```diff
@@ -224,6 +224,14 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
     );
   }
 
+  _isFirefox() {
```

Comment:
**If** this is needed at all, it does not belong into an individual component, but into a utility file outside.